omnigibson/object_states/toggle.py

```diff
@@ -41,30 +47,58 @@ def _set_value(self, new_value):
     def _initialize(self):
         super()._initialize()
         self.initialize_link_mixin()
-        mesh_prim_path = f"{self.link.prim_path}/visual_marker"
+        mesh_prim_path = f"{self.link.prim_path}/mesh_0"
+        pre_existing_mesh = get_prim_at_path(mesh_prim_path)
         # Create a primitive mesh if it doesn't already exist
-        if not get_prim_at_path(mesh_prim_path):
+        if not pre_existing_mesh:
```

Comment:
hmm why are we creating a sphere mesh here rather than a sphere shape? I thought the latter will be better for consistency.

Reply:
whoops! great point. fixed